### PR TITLE
Feature/SEEXT-10852

### PR DESCRIPTION
**Pull request by sstimac**
- Fix modal crash on animating
- Fix button icon animating in wrong way

### CODE REVIEW

File: components/YearRangePicker/YearRangePickerModal.js [hunk 1]
```diff
@@ -221,6 +236,7 @@ class YearRangePickerModal extends PureComponent {
         onBackdropPress={onDismiss}
         useNativeDriver
         backdropOpacity={Platform.OS === 'ios' ? 0.7 : 0}
+        animationIn="slideInUp"
```
Comment (Definitely-Not-Vlad) [inline components/YearRangePicker/YearRangePickerModal.js:239]: Just a note, if you add 1 line to a file and in your PR it has 96 LoC changed, separate autofix changes into a separate PR, please. With 1 file it's fine, but if that happens with 4-5 or however many files, it can waste a _lot_ of time for nothing.

**Review by Definitely-Not-Vlad — APPROVED** [review]

**Review by ltatarev — APPROVED** [review]

**Review by tomislav-arambasic — APPROVED** [review]
